src/ISADotnet/ISADotNet.fsproj

```diff
@@ -1,12 +1,15 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
+
   <ItemGroup>
-    <Compile Include="JsonExtensions.fs" />
```

Comment:
Any reason for changing the compile order?

Reply:
Brought compile order more in line with previous version!